youngseo/app.js

```diff
@@ -0,0 +1,44 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+
+// 환경 변수 선언 이전에 설정 
+dotenv.config()
+
+const { DataSource } = require('typeorm');
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+})
+
+myDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized!")
+    })
+
+const app = express()
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan('dev'));
+
+
+app.get("/ping", (req,res) => {
+    res.json({ message : "pong"})
+});
```

Comment:
`/ping` 이라는 엔드포인트로 구현되는 아주 간단한 API는 heath check (서버가 제대로 작동 중인지 최소한의 확인을 하기 위한) 용도로 사용 됩니다. 현재 의도한 바로는 request가 들어 올 때 pong이라는 메세지를 client에게 return 해주는 것인데 pong이 리턴이 됐다는 것은 API에 대한 요청이 성공적으로 이루어졌다는 것을 의미하고 이에 대한 적절한response code가 같이 반환 돼야 할 필요가 있습니다!

```suggestion
app.get("/ping", (req, res) => {
  return res.status(200).json({ message: "pong" });
});
```